### DISCUSSION TITLE
wiki/PredatorPrey_step8

### DISCUSSION THREAD

**Discussion by giscus** [gama-platform/gama-platform.github.io#207] [General]
# wiki/PredatorPrey_step8

This eighth step illustrates how to define more complex actions, how to use conditional statements and iterator operators over containers.

https://gama-platform.org/wiki/PredatorPrey_step8

**Comment by mbalagueraj** [comment]
Maybe a silly question. Could somebody tellme the meaning of the prefix "nb_" attached to some global variables?, thanks.

**Reply by RoiArthurB** [reply]
Number :)

**Reply by mbalagueraj** [reply]
Thanks, a greeting.
